src/components/account/LoginCard.tsx

```diff
@@ -28,24 +31,32 @@ interface Props {
 }
 
 export default function LoginCard(props: Props): ReactElement {
-  const onFinish = () => {
-    alert('Finish');
-    //console.log('Success:', values);
-    // TODO https://promopal.atlassian.net/jira/software/projects/PP/boards/1?selectedIssue=PP-42
-    // TODO https://promopal.atlassian.net/jira/software/projects/PP/boards/1?selectedIssue=PP-43
+  const firebase = useFirebase();
+  const history = useHistory();
+
+  const onFinish = (data: { email: string; password: string; staySignedIn: boolean }) => {
+    firebase
+      .doSignInWithEmailAndPassword(data.email, data.password, data.staySignedIn)
```

Comment:
This is kind of reminder. As we discussed before, the user have to request `/users/firebase/:firebaseId` after the user succeed to sign in by firebase.

Reply:
@yunishimura0716 I will be working on the sign in / sign up auth flow in new PR connecting it to the BE and following your auth flow doc!